src/modules/SearchPage/components/MoveView/components/MoveCard.tsx

```diff
@@ -0,0 +1,19 @@
+import { AvailableCardDetails, PokemonMoveDetails } from 'types';
+import { useNavigate } from 'react-router-dom';
+import * as S from '../styled';
+
+type Props = {
+  move: PokemonMoveDetails;
+};
+
+export const MoveCard = ({ move }: Props) => {
+  const navigation = useNavigate();
+  const onClick = () => {
```

Comment:
handleCardClick sounds much better, but up to you

Reply:
Yeah, changed